cmd/minishift/cmd/start.go

```diff
@@ -677,6 +681,30 @@ func initSubscriptionManagerFlags() *flag.FlagSet {
 	return subscriptionManagerFlagSet
 }
 
+// Get config from the start flags and set it to viper config so that in
+// stop start case user don't need to remember the start flags.
+// Note: This is only going to work for start flag not cluster up flags.
+func populateStartFlagsToViperConfig() {
+	startFlagSet.VisitAll(func(flag *flag.Flag) {
+		if viper.IsSet(flag.Name) {
```

Comment:
I guess we should also log this when repopulating, WDYT?

Reply:
@gbraad sure, you mean to put a print statement that repopulating the startup flags?